integration_test/support/models.exs

```diff
@@ -173,3 +175,17 @@ defmodule Ecto.Integration.Tag do
     field :uuids, {:array, Ecto.UUID}
   end
 end
+
+defmodule Ecto.Integration.Profile do
+  @moduledoc """
+  This module is used to array
+
+    * Embedding
+
+  """
+  use Ecto.Integration.Model
+
+  schema "" do
```

Comment:
Nah, it is fine. :) Just give whatever name and we will document it is simply not used when embedded.